app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,33 @@ def show
     puts params
     @post = Post.find(params[:id])
   end
+
+  # form actions
+  def new
+    @user = User.find(params[:user_id])
+    @post = @user.post.new
+    render :new, locals: { post: @post }
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.post.new(post_params)
+    respond_to do |format|
+      format.html do
+        if @post.save
+          flash[:success] = 'Post created successfully'
+          redirect_to user_posts_path(@user)
```

Comment:
- [ ] If successful this should render the @post that was just created not the user. I think it will be more user friendly